roadmap/roadmap.md

```diff
@@ -30,59 +30,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
+- :large_orange_diamond: Improve and/or maintain interoperability with performant I/O file formats and libraries such as HDF5 and Dask.
 
-- :red_square: Improve and/or maintain interoperability with performant I/O file formats and libraries such as HDF5 and Dask.
+- :red_square: Improve support for using Astropy tools in heterogeneous computing environments such as cloud environments or GPU systems.
 
-- :red_square: Improve support for using Astropy tools in high-performance machine learning and data analysis frameworks, such as on GPU systems.
+## Learn and User Support
 
-## Learn and User Support 
+- :green_circle: Generate and ingest guides and/or a series of tutorials that demonstrate Astropy Project functionality in the context of astronomical research.
 
-- :green_circle: Improve discoverability of documentation and educational materials by overhauling the Learn website, including implementing an integrated search functionality and building an example gallery.
+-  :large_orange_diamond: Update Astropy website: Rework the Team page and improve the visibility of Astropy Learn.
 
 - :large_orange_diamond: Expand and build on workshop offerings by recruiting and training more facilitators and expanding the geographic diversity of meetings where we offer workshops. 
 
-- :green_circle: Expand the number and diversity of user support resources, including establishing a user-focused forum on Discourse.
+- :large_orange_diamond: Expand the number and diversity of active user support resources.
 
-- :large_orange_diamond: Generate and ingest guides and/or a series of tutorials that demonstrate Astropy Project functionality in the context of astronomical research, especially focusing on spectroscopy tasks.
+- :red_square: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
 
-- :red_square: Develop tutorials suitable for use in university astronomy courses. 
+- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses. 
 
 ## Community Building and Sustainability
 
 - :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors.
 
-- :large_orange_diamond: Increase inclusion, diversity, and empowerment efforts within the Astropy Project and NumFOCUS communities, and improved our understanding the demographics of our communities in order to measure the effectiveness of these efforts.
-
 - :large_orange_diamond: Better understand Astropy user community through a NumPy-like user survey (see [this summary](https://numpy.org/user-survey-2020-details/) of their results).
 
+- :red_square: Increase inclusion, diversity, and empowerment efforts within the Astropy Project and NumFOCUS communities, and improved our understanding the demographics of our communities in order to measure the effectiveness of these efforts.

```

Comment:
and improve our understanding of the

Reply:
"...and improve our understanding of the..."

Also need the extra "of" after understanding.